Nimble Templates/App.xctemplate/TemplateInfo.plist

```diff
@@ -16,6 +16,167 @@
 	<string>This template provides a starting point for an application that uses a single view.</string>
 	<key>SortOrder</key>
 	<integer>1</integer>
+	<key>Project</key>
+	<dict>
+		<key>Configurations</key>
+		<dict>
+			<key>Dev Staging</key>
+			<dict>
+				<key>ONLY_ACTIVE_ARCH</key>
+				<string>YES</string>
+				<key>GCC_OPTIMIZATION_LEVEL</key>
+				<string>0</string>
+				<key>GCC_PREPROCESSOR_DEFINITIONS</key>
+				<string>DEBUG=1 STAGING=1 $(inherited)</string>
+				<key>MTL_ENABLE_DEBUG_INFO</key>
+				<string>INCLUDE_SOURCE</string>
+				<key>ENABLE_TESTABILITY</key>
+				<string>YES</string>
+				<key>GCC_DYNAMIC_NO_PIC</key>
+				<string>NO</string>
+				<key>SWIFT_OPTIMIZATION_LEVEL</key>
+				<string>-Onone</string>
+				<key>SWIFT_ACTIVE_COMPILATION_CONDITIONS</key>
+				<array>
+					<string>DEBUG</string>
+					<string>STAGING</string>
+				</array>
+				<key>DEBUG_INFORMATION_FORMAT</key>
+				<string>dwarf</string>
+				<key>PRODUCT_BUNDLE_IDENTIFIER</key>
```

Comment:
The attribute `PRODUCT_BUNDLE_IDENTIFIER` does not work, it create duplicate values, please check again.